### PR TITLE
[2.13.x] DDF-04415 Solr stop key for windows

### PR DESCRIPTION
**Pull request by ahoffer**
#### What does this PR do?
Generates a stop key every time Solr is started by the `ddfsolr.bat` script.

#### Who is reviewing it? 
@garrettfreibott  @mcalcote @brjeter @tyler30clemens @austinsteffes 

#### Select relevant component teams: 
@codice/security 
@codice/solr 
#### Ask 2 committers to review/merge the PR and tag them here.
@brjeter 
@garrettfreibott 

For GH Issues:
Fixes: #4415 

####Testing
* Testing is limited to Windows
* Test `ddfsolr.bat start`,  `ddfsolr.bat restart`,  and `ddfsolr.bat stop`
* Test that `ddfsolr.bat stop` removes the file named STOPKEY
                  
#### Checklist:
- [No] Documentation Updated
- [No] Update / Add Threat Dragon models
- [No] Update / Add Unit Tests
- [No] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/ddf-common/src/main/resources/bin/ddfsolr.bat [hunk 1]
```diff
@@ -35,17 +38,29 @@ IF "!solr.http.protocol!"=="https" (
 IF "%COMMAND%"=="" ECHO Missing command. Use start, restart, stop.
 
 IF "%COMMAND%"=="start" (
+    CALL :GEN_STOPKEY
     IF "!solr.http.protocol!"=="http" ECHO **** USING INSECURE SOLR CONFIGURATION ****
     CALL %SOLR_EXEC% start -p !solr.http.port! -m !solr.mem!
 )
 
 IF "%COMMAND%"=="restart" (
+    SET /P key key=<STOPKEY
```
Comment (brjeter) [inline distribution/ddf-common/src/main/resources/bin/ddfsolr.bat]: ❓ What is this line doing?

Comment (ahoffer) [reply distribution/ddf-common/src/main/resources/bin/ddfsolr.bat]: That does not look right.

**Comment by tyler30clemens** [comment]
Hero Successful ✅
Was able to run the three commands without issues. Verified file named STOPKEY appeared at `start` and was removed after executing `stop`

File: distribution/ddf-common/src/main/resources/bin/ddfsolr.bat (hunk 1 above)
Comment (ahoffer) [reply distribution/ddf-common/src/main/resources/bin/ddfsolr.bat]: fixed

**Comment by brjeter** [comment]
Sidestepped CI, as these files do not affect the CI builds.

**Pull request merged**